src/components/pages/Offer/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -279,6 +280,15 @@ const OfferForm = ({
     return fieldName in formErrors ? formErrors[fieldName] : null
   }
 
+  const canSelectedTypeBeOfflineButOnlyVirtualVenues = useCallback(() => {
```

Comment:
```suggestion
  const hasPhysicalVenues = useCallback(() => {
```

Du coup cette méthode s'occuperait uniquement de vérifier si la structure a des lieux physiques
Et le reste de la condition est plus bas